lib/init/parser.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const parser = require('body-parser');
```

Comment:
This is internal to Passport. Remember the bug last friday? Yeah...that's why this is here.

Without it, passport silently, painfully silently, fails.